src/coreclr/vm/clsload.cpp

```diff
@@ -769,19 +769,10 @@ TypeHandle ClassLoader::LoadConstructedTypeThrowing(TypeKey *pKey,
 
     // Lookup in the classes that this class loader knows about
 
-    if (pKey->HasInstantiation() && ClassLoader::IsTypicalSharedInstantiation(pKey->GetInstantiation()))
-    {
-        _ASSERTE(pKey->GetModule() == ComputeLoaderModule(pKey));
-        typeHnd = pKey->GetModule()->LookupFullyCanonicalInstantiation(pKey->GetTypeToken(), &existingLoadLevel);
-    }
-
-    if (typeHnd.IsNull())
+    typeHnd = LookupTypeHandleForTypeKey(pKey);
```

Comment:
Move the declaration of `typeHnd` to this line too?

Reply:
sure.